.github/workflows/verify-docker-image.yml

```diff
@@ -11,42 +11,80 @@ jobs:
     strategy:
       matrix:
         include:
-          - expected: { version: local, push: false }
-            input: { version: '', push: '' }
-          - expected: { version: version, push: true }
-            input: { version: version, push: true }
+          - expected: { version: 'local', push: false, uid: ''}
+            input: { version: '', push: '', uid: '' }
+          - expected: { version: version, push: true, uid: '1' }
+            input: { version: version, push: true, uid: '1' }
 
     steps:
       - uses: actions/checkout@v4
 
       - name: Check Docker Compose (default)
         id: default
-        env:
-          DOCKER_VERSION: ${{ matrix.input.version }}
-          DOCKER_PUSH: ${{ matrix.input.push }}
         shell: bash
         run: |
-          set -xue
-          config=$(make docker_compose_config)
+          set -ue
 
-          # Expect image tag is correct
-          echo $config | grep -q "image: mozilla/addons-server:${{ matrix.expected.version }}"
+          export SUPERUSER_EMAIL="email"
```

Comment:
I could probably reimplement this in python... wdyt?

Reply:
Probably worth extracting somewhere (in Python or not) cause there is a lot going on and it's hard to read...

Reply:
I've extracted the exiisting bash test, and the new tests introduced here to a JS `make.spec.js` 

The reason for JS is that the test MUST run on the host, it relies on docker commands and we don't (probably don't want to) install docker in our container. no need for the bloat.

Theoretically I could write it in python, but I felt it easier to run the js tests on the host.